subsys/bluetooth/host/Kconfig.l2cap

```diff
@@ -70,4 +70,12 @@ config BT_L2CAP_SEG_RECV
 	  This API enforces conformance with L2CAP TS, but is otherwise as
 	  flexible and semantically simple as possible.
 
+config BT_L2CAP_RECONFIGURE_EXPLICIT
+	bool "L2CAP Explicit reconfigure API [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+
```

Comment:
minor: there's normally no empty line between "help" and the actual help text.

Reply:
ah, I see this is a likely copy-paste from the `BT_L2CAP_SEG_RECV` definition. Both should be fixed.

Reply:
Sorry. I just have a habit of forgetting to remove the blank line after I add it to re-wrap the text with my editor. Wish we had a dedicated formatting tool for Kconfig.